docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -133,6 +133,7 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | agent-analyzer | default | Agent Analyzer. | SW_AGENT_ANALYZER | default |
 | - | -| sampleRate|Sampling rate for receiving trace. The precision is 1/10000. 10000 means 100% sample in default.|SW_TRACE_SAMPLE_RATE|10000|
 | - | - |slowDBAccessThreshold|The slow database access thresholds. Unit ms.|SW_SLOW_DB_THRESHOLD|default:200,mongodb:100|
+| - | - |forceSampleErrorSegment|When sampling mechanism active, this config can open(true) force save some error segment. true is default.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
```

Comment:
```suggestion
| - | - |forceSampleErrorSegment|When sampling mechanism activated, this config would  make the error status segment sampled, ignoring the sampling rate.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
```

Reply:
And change the sampling doc when same way. The previous version has some grammar issue.